### PR TITLE
Classiq Submission

### PR DESCRIPTION
**Pull request by LisithaDiz**
Classiq Submission - 1
Classiq Submission - 2

### CODE REVIEW

**Comment by LisithaDiz** [comment]
VQE HW 3

**Comment by LisithaDiz** [comment]
HW

File: community/QClass_2024/Submissions/HW3/DM_Lisitha_Dissanayaka_HW3_QClass2024.ipynb [hunk 1]
```diff
@@ -0,0 +1,437 @@
+{
```
Comment (Nadav138) [inline community/QClass_2024/Submissions/HW3/DM_Lisitha_Dissanayaka_HW3_QClass2024.ipynb:1]: <p>Hi!</p><p>Seems to have some problem here. </p><p>In the second case, "Optimal energy" should be around : -1.8... as the electronic energy in class. </p><p>check the solution when it's published :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

**Review by Nadav138 — APPROVED** [review]

**Comment by Nadav138** [comment]
Please open new PR, as this one failed in the tests. 
Make sure you open the new PR from a new and updated branch.
Thanks and well done for the work!

**Pull request closed without merging**